tests/cases/fourslash/getOutliningForObjectsInArray.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path="fourslash.ts"/>
+
+////// objects in x should generate outlining spans that do not render in VS

```

Comment:
The particular point about the spans not rendering in VS is not validated, but the placement of the spans is. This test would have failed before my change.